pennylane/ops/qubit.py

```diff
@@ -253,6 +278,11 @@ def _matrix(cls, *params):
     def _eigvals(cls, *params):
         return np.array([1, cmath.exp(1j * np.pi / 4)])
 
+    @staticmethod
+    def decomposition(wires):
+        decomp_ops = [RZ(np.pi / 4, wires=wires)]
```

Comment:
As with the above case, this could also be done without the global phase factor via `qml.PhaseShift(np.pi / 4, wires=wires)`